imports/startup/client/routes.js

```diff
@@ -102,7 +102,7 @@ FlowRouter.route('/playlists', {
   }
 });
 
-FlowRouter.route('/playlists/:id', {
+FlowRouter.route('/playlist/:id', {
```

Comment:
Thanks. You can view the whole history of all playlists as one big 'playlist', or anyway that's how i've thought of it.